zipline/finance/performance/period.py

```diff
@@ -401,6 +435,8 @@ def as_account(self):
                     self.ending_cash + self.ending_value)
         account.total_positions_value = \
             getattr(self, 'total_positions_value', self.ending_value)
+        account.total_positions_value = \
```

Comment:
@brianpfink why are these all `getattr` calls instead of just `self.attribute`.?